### ISSUE TITLE
SDK v2.1 charge does not work

### ISSUE THREAD

**Issue by anddybdahl** [iZettle/sdk-ios#246]
# Expected Behavior
When the charge function is called, a payment flow is expected to open.

# Current Behavior

What is the current behavior?
When the charge function is called, a payment flow is expected to open. Instead a screen opens and displays "Something went wrong."

# Steps to Reproduce

Please provide detailed steps for reproducing the issue.
1. The SDK is started in AppDelegate with an API key.
2. In A ViewController, after presenting the settings screen, logging in and connecting an iZettle reader 2, the charge function is called.
3. A screen opens, displaying a "Something went wrong" message.

* Operating Version:
* SDK version: v2.1

## Failure Logs

Please include any relevant log snippets or files here.

**Comment by valkovsky** [comment]
Hi @anddybdahl !

Could you provide more details to help us reproduce the issue:
- reader software version (it could be checked it settings screen)
- iOS version
- device type (iPhone or iPad)
- also screenshot of screen with `Something went wrong` error
- what account do you use? is it demo account that we provide or real account? 

Best regards,
Andrei Valkovskii

**Comment by anddybdahl** [comment]
Hi @valkovsky ,
I tried an older version yesterday (2.0.5), and that showed an error letting me know that the grants where wrong (the app had a different bundle identifier than our registered app, so thats what the error let me know)
In the new version, only "Something whent wrong" is shown, so I think its just the error message that have changed from v2.0.5 to v2.1. Would be nice if you fixed the newest version to show concise error messages for faster debugging though.

I tested on
IOS 10.3
iPhone 5
I was using the demo account, but as I mentioned above, a wrong bundle identifier.
